src/masternodeman.h

```diff
@@ -96,7 +96,7 @@ class CMasternodeMan
             READWRITE(strVersion);
         }
 
-        READWRITE(vMasternodes);
+        READWRITE(mapMasternodes);
```

Comment:
Wondering why CMasternodeMan needs to be serialized. mncache.dat?
If it's really serialized, this may result in incompatibility.

Reply:
Good point! I'll bump `CMasternodeMan::SERIALIZATION_VERSION_STRING`